src/web/app/pages-instructor/instructor-course-edit-page/instructor-edit-panel/instructor-edit-panel.component.html

```diff
@@ -72,15 +72,17 @@
         <input id="checkbox-display-instructor-{{ instructorIndex + 1 }}" checked=""
                type="checkbox" [ngModel]="instructor.isDisplayedToStudents"
                (ngModelChange)="triggerModelChange('isDisplayedToStudents', $event)" [disabled]="!instructor.isEditing">
+        &nbsp;
         <span ngbTooltip="If this is unselected, the instructor will be completely invisible to students. E.g. to give access to a colleague for ‘auditing’ your course">Display to students as:</span>
       </label>
       <div class="col-sm-9">
         <input id="displayed-name-instructor-{{ instructorIndex + 1 }}" class="form-control"
                type="text"
                (ngModelChange)="triggerModelChange('displayedToStudentsAs', $event)"
                [disabled]="!instructor.isDisplayedToStudents || !instructor.isEditing"
-               placeholder="E.g.Co-lecturer, Teaching Assistant"
-               [ngModel]="instructor.isDisplayedToStudents ? instructor.displayedToStudentsAs : '(This instructor will NOT be displayed to students)'">
+               [ngModel]="instructor.isDisplayedToStudents ? instructor.displayedToStudentsAs : '(This instructor will NOT be displayed to students)'"
+               [defaultValue]="'Instructor'"
```

Comment:
Question here: Will the actual model get updated? This looks magic to me unless the event `triggerModelChange('displayedToStudentsAs', $event)` get fired somehow by the property.

Reply:
Yeap, I tested this by creating a new instructor and watching the http request payload for the api call. The default value does get inserted in if the user leaves it blank.

Reply:
Although it works, I would prefer to change the default value in `newInstructorPanel: InstructorEditPanel`. 

Actually, if the initial value added in `newInstructorPanel: InstructorEditPanel` will be overriden by `[defaultValue]`. We have a problem as the template html should not store the modal data.

Reply:
I think the setting of `defaultValue` triggers `ngModelChange` but I can't find supporting references for this so it's just my guess.